.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,121 @@
+name: Benchmark CI
+
+on:
+  pull_request:
+    branches:
+      - meson
+      - master
+    # types: [labeled]
+
+  # workflow_dispatch:
+
+env:
+  CCACHE_DIR: "${{ github.workspace }}/.ccache"
+  INSTALLDIR: "installdir"
+
+jobs:
+  benchmark:
+    name: Benchmark
+    # if: ${{ github.event.label.name == 'run_benchmark' && github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' }}
```

Comment:
Uncomment, and add the protections for not running on a fork (see the `linux_meson` job)?